sdk/eventhub/Azure.Messaging.EventHubs.Processor/README.md

```diff
@@ -35,14 +35,16 @@ To quickly create the needed resources in Azure and to receive connection string
 Install the Azure Event Hubs Event Processor client library for .NET using [NuGet](https://www.nuget.org/):
 
 ```PowerShell
-Install-Package Azure.Messaging.EventHubs.Processor -Version 5.0.0
+dotnet add package Azure.Messaging.EventHubs.Processor
```

Comment:
FWIW - I think that by default this will not install a pre-release build, and you have to pass `--version 5.1.0-preview.1`

Reply:
Not sure if it is better to call this out in the release notes (i.e. to install this pre-release build, do this special thing) or update the general guidance here.

Reply:
Part of the doc review guidance from our PM team was to remove the explicit version numbers.   AFAIK, we're aiming to have the read me provide guidance on pulling the latest GA package and not explicitly target previews... but @czubair would be the authoritative voice.